### ISSUE TITLE
Pretrain model

### ISSUE THREAD

**Issue by h-aboutalebi** [minniie/multimodal_chat#5]
Hi,

I am reproducing your results. I was wondering if you can share the pretrained model for response generation?
Otherwise, I cannot get the results you reported in the thesis.

**Comment by minniie** [comment]
Hi @h-aboutalebi, thank you for your interest in my paper.

Unfortunately, I no longer have the checkpoints for the finetuned models, so I would need to retrain them myself.
Can you tell me the specific configuration you used for training and the metrics you cannot reproduce from the paper?
I will take a look and compare the results with mine.

Thank you :)

**Comment by h-aboutalebi** [comment]
Thanks for responding back. 
First, I do not have the source code for Divter so it is not clear to me how I can replicate the results you put.
Second, there is not much information on how you choose and change the learning rate during the training. You have put different models in the paper for response generation but I think the current code only provides the results for DialoGPT-medium. Can you provide instructions for the rest?

**Comment by minniie** [comment]
Hi @h-aboutalebi,

1. The scores for Divter are taken from [their original paper](https://arxiv.org/abs/2110.08515), as mentioned in pg. 37 of [my paper](https://arxiv.org/abs/2305.03512).

2. I use the default config in transformers [TrainingArguments](https://huggingface.co/docs/transformers/v4.26.1/en/main_classes/trainer#transformers.TrainingArguments), also as mentioned in [my paper](https://arxiv.org/abs/2305.03512). For your information, the default learning rate scheduling is linear decay with initial learning rate of 5e-5. This scheduling is applied to all response generator models. You can easily run the code with pretrained models other than DialoGPT-medium if you change the arguments below. 
https://github.com/minniie/multimodal_chat/blob/9b7cc7b8da7b68dcffc78c41e8bd48b1086f0d89/sh/train_response_generator.sh#L5-L6

Let me know if you have any other questions!
